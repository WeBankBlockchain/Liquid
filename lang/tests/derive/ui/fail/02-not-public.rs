use liquid_lang as liquid;
use liquid::InOut;

#[derive(InOut)]
struct MyStruct {
    i: i128,
}

fn main() {}
