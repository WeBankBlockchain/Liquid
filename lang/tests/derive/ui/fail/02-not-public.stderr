error: the visibility of this type should be `pub`
 --> $DIR/02-not-public.rs:5:1
  |
5 | / struct MyStruct {
6 | |     i: i128,
7 | | }
  | |_^
